include/swift/SwiftRemoteMirror/SwiftRemoteMirrorTypes.h

```diff
@@ -248,6 +248,9 @@ typedef struct swift_async_task_info {
   uint8_t IsRunning;
   uint8_t IsEnqueued;
 
+  uint8_t HasThreadPort;
```

Comment:
I assume that the only reason that we need `HasThreadPort` is because `ThreadPort` is 32-bit and `mach_port_t` is `uintptr_t` sized, and thus we cannot use the traditional invalid port specifier?